graphql-client/src/main/graphql/client.graphql

```diff
@@ -397,6 +397,47 @@ query Infrastructures(
     }
 }
 
+#ph########### Session ############
+
+fragment SessionPart on Session {
+    id
+    secret
+    producerBindings {
+        ... ProducerBindingPart
+    },
+    consumerBindings {
+        ... ConsumerBindingPart
+    }
+    expiresAt
+}
+
+mutation InsertSession (
```

Comment:
`CreateSession` for consistency with the actual method name

Reply:
👍